docs/TFM/projectTool/docusaurus.md

```diff
@@ -1,3 +1,30 @@
+<<<<<<< HEAD
```

Comment:
This is bad sign.. looks like you have a merge conflict you did not resolve correctly.. 
Please review and make sure the file looks good at you local tree.
Then make sure its all committed and push again to your brnach